lib/python/qmk/cli/doctor.py

```diff
@@ -16,8 +16,8 @@
     'dfu-programmer': {},
     'avrdude': {},
```

Comment:
```suggestion
    'avrdude': {'version_arg': '-?'},
```

Reply:
The option `-?` doesn't work on zsh it seems. Works fine on bash though. `-\?` works for both. So set the arg to `-\\?`

Reply:
No, it should just be `-?`, even on zsh. If I print the output of avrdude, it gives me this otherwise:
```
INFO Found avrdude version C:\msys64\home\fauxpark\qmk_utils\avr8-gnu-toolchain\bin\avrdude.exe: invalid option -- \
[...]
```
It probably only needs to be escaped when being run in an interactive terminal environment.